mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -1008,7 +1008,10 @@ func expandLinuxNodeConfig(v interface{}) *container.LinuxNodeConfig {
 	if len(ls) == 0 {
 		return nil
 	}
-	cfg := ls[0].(map[string]interface{})
+	cfg, ok := ls[0].(map[string]interface{})
+	if !ok {
+		return nil
+	}
```

Comment:
```suggestion
	if ls[0] == nil {
		return &container.LinuxNodeConfig{}
	}
	cfg := ls[0].(map[string]interface{})
```
Can we try something like this and see if the API will return back the value?

Reply:
Thank you for your assistance! You are right; this change worked perfectly.